stats/release_queue.php

```diff
@@ -152,23 +152,23 @@
     }
 
 
-    $comments_url1 = mysqli_real_escape_string(DPDatabase::get_connection(), "<a href='$code_url/project.php?id=");
-    $comments_url2 = mysqli_real_escape_string(DPDatabase::get_connection(), "'>");
-    $comments_url3 = mysqli_real_escape_string(DPDatabase::get_connection(), "</a>");
+    $comments_url1 = DPDatabase::escape("<a href='$code_url/project.php?id=");
+    $comments_url2 = DPDatabase::escape("'>");
+    $comments_url3 = DPDatabase::escape("</a>");
 
     dpsql_dump_themed_query("
         SELECT
 
             concat('$comments_url1',projectID,'$comments_url2', nameofwork, '$comments_url3') as '" 
-                . mysqli_real_escape_string(DPDatabase::get_connection(), _("Name of Work")) . "',
-            authorsname as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Author's Name")) . "',
-            language    as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Language")) . "',
-            genre       as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Genre")) . "',
-            difficulty  as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Difficulty")) . "',
-            username    as '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Project Manager")) . "',
+                . DPDatabase::escape(_("Name of Work")) . "',
```

Comment:
we can't use sprintf here?

Reply:
True. There's a bunch of unwieldy concats here, but the main problem with `sprintf` is that it makes it harder on the reader to check which `%s` matches up with which `sprintf` argument. If only PHP format specifiers could take named arguments like Python/Rust can.

Reply:
Yes, let's not try to fix this with `sprintf()` here since we need to just stop trying to squeeze direct table output. I aspire to rip all the `dpsql_dump_*()` functions out at some point.

What you have here is good, @bpfoley.